src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -0,0 +1,67 @@
+import math
+
+def get_lot_size(smallest_decimal_unit: float, coin_decimals: int) -> int:
```

Comment:
How about `get_lot_size_integer` to clarify that this is outputting the integer value to use for market registration?